src/SelfAssessment/EmergencyAssessment.js

```diff
@@ -42,17 +36,21 @@ export const EmergencyAssessment = ({ navigation }) => {
         <ChoiceButtons
           agreePress={handleAgreePress}
           agreeTitle={
-            <TranslationButtonText
-              translator={t}
-              text={"assessment.agree_option_agree"}
-            />
+            <RTLEnabledText style={styles.boldText}>
+              {t("assessment.i_am") + " "}
```

Comment:
This is because there's a rule about not rendering text outside a `Text` component but since this is not syntactically a `Text` component `eslint` is not happy about it. I guess I could put this in a variable outside, happy to do so.

Reply:
ive been using consts to get around this, but maybe we want to remove the eslint rule?